Makefile

```diff
@@ -86,7 +86,7 @@ uninstall: manifests kustomize  ## Uninstall CRDs from a cluster
 configure-operator-webhook:
 	ENABLE_OPERATOR_WEBHOOK=$(ENABLE_OPERATOR_WEBHOOK) hack/configure_operator_webhook.sh
 
-deploy-cert-manager:
+deploy-cert-manager: ## Deploys cert-manager from online
```

Comment:
I'd write fetching the cert0manager from https://github.com/jetstack/cert-manager

Reply:
fixed